solutions/community-deploy/cloud/AWS/README.md

```diff
@@ -48,66 +43,96 @@ os_version          = "CentOS 7.9.2009 x86_64"
 region              = "ap-south-1"
 security_group_cidr = "134.204.222.36/32"
 key_name            = "devops-key"
```

Comment:
Please add ebs_volume_count , instance_count  , ebs_volume_size and tag_name  in this table. Somehow it's not showing the details  in https://github.com/Seagate/cortx-re/tree/mukul-multinode-2/solutions/community-deploy/cloud/AWS 

![image](https://user-images.githubusercontent.com/63289460/182360881-994c0514-3850-45f7-87f7-a6ea9a619392.png)